WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -89,6 +89,25 @@ + (void)disableScreenshots
   [[NSUserDefaults standardUserDefaults] setBool:YES forKey:@"DisableScreenshots"];
 }
 
++ (void)enableScreenshots
+{
+  [[NSUserDefaults standardUserDefaults] setBool:NO forKey:@"DisableScreenshots"];
+}
+
++ (void)disableScreenshotsUnlessEnvironment
+{
+  BOOL disable = YES;
+  NSString * envSetting = NSProcessInfo.processInfo.environment[@"DISABLE_SCREENSHOTS"];
```

Comment:
This might be confusing. How about changing the variable name to `ENABLE_SCREENSHOTS` and actually enable them if this variable is defined without checking its actual value?

Reply:
I would also rename the env variable itself to make its name more descriptive: ENABLE_AUTOMATIC_SCREENSHOTS